### ISSUE TITLE
Resnet ImageNet Ckpt missing

### ISSUE THREAD

**Issue by sandwichhlord** [McCrearyD/rigl-torch#9]
First off, thank you for putting together such a clean and robust PyTorch reproduction of RigL! I am a B.Tech CSE student currently conducting a topological ablation study on dynamic sparse networks, and your implementation is exactly what I need for my baseline comparison.

I was trying to download the the ResNet-50 RigL-1x ImageNet result ckpt in the README.md file but it gives a broken link error. 

If you are able to re-upload it or attach it to a GitHub Release, it would be a massive help for my research and save me from having to run a highly expensive GPU training job from scratch just to extract the topological masks.

Thank you for your time! Finding this open-source project was really a huge relief of my shoulders!

Best regards,
Avi Khurana

**Comment by verbiiz** [comment]
Hi, thanks for reaching out! I wish I still had the model file, but I searched everywhere and couldn't locate them anywhere. It shouldn't be too expensive if you don't mind waiting a full day to train. I believe if you run the train script on a single V100, it should take maybe 24 hours to train on ImageNet? The only problem is you would have to download the dataset to the machine which is about 250gb but it shouldn't take long in the grand scheme of things.

From what I recall the script on the main branch is stable, but do double check with some smaller subsets before launching a train job if you still wanted to use rigl-torch for your benchmarks. 

Feel free to comment on this issue with any problems you face and I'd be happy to help! If you do end up training a 90% sparse ResNet50 and are willing to share the weights, do let me know and I'd be happy to put you in an acknowledgements section as well as update the download link on the readme.

Thanks for raising this! I didn't realize those files vanished from my google drive. Woops! Sorry about that.

**Comment by verbiiz** [comment]
Oh and if you use a pretrained model to start, the training to 90% sparse should only take mabe an hour on a V100. If I recall correctly. So if you're looking to save time- definitely start with a pretrained model and take note of accuracy / sparsity.

**Comment by sandwichhlord** [comment]
Hey! Thanks for letting me know!

Its very unfortunate that the files got wiped, I'm sorry about that.
I've decided to train a smaller resnet18 model on just the CIFAR10 dataset as I dont have access to a lot of hardware, but hopefully my college pulls through and allows me to use theirs in the near future.

Thank you!
